test/fn/watch-project.spec.js

```diff
@@ -273,6 +275,55 @@ describe('watch-project', function () {
       });
   });
 
+  it('watch-project: convert training forms', () => {
+    const form = 'new_actions_training';
+    const copyForm = () => copySampleForms('training-xlsx', TRAINING_FORMS_PATH);
+
+    return watchWrapper(copyForm, `${form}.xlsx`)
+      .then(() => {
+        const forms = fs.fs.readdirSync(trainingFormDir);
+        expect(forms).to.include(`${form}.xml`);
+      })
+      .then(() => cleanFormDir(trainingFormDir, form));
```

Comment:
Is it possible to move `cleanFormDir` in an `afterEach` hook? I would expect that each test should start with an empty form directory.

Reply:
Done